### ISSUE TITLE
SA1206 false positive for ref struct

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2546]
The following C# 7.2 construct is resulting in a SA1206 false positive:

```csharp
private ref struct BitHelper
{
}
```

**Comment by pdelvo** [comment]
I take this one.

**Comment by sm-g** [comment]
Same for `readonly public struct`
